test/AuthenticationRequest.request.spec.ts

```diff
@@ -110,9 +128,17 @@ describe('create Request Uri should', () => {
         kid: 'did:key:z6MkixpejjET5qJK4ebN5m3UcdUPmYV4DPSCs1ALH8x2UCfc#keys-1',
       },
       registration: {
-        didMethodsSupported: ['did:ethr:'],
-        subjectIdentifiersSupported: SubjectIdentifierType.DID,
-        credentialFormatsSupported: CredentialFormat.JSON_LD,
+        idTokenSigningAlgValuesSupported: [SigningAlgo.EDDSA, SigningAlgo.ES256],
+        requestObjectSigningAlgValuesSupported: [SigningAlgo.EDDSA, SigningAlgo.ES256],
+        responseTypesSupported: [ResponseType.ID_TOKEN],
+        scopesSupported: [Scope.OPENID_DIDAUTHN, Scope.OPENID],
+        subjectSyntaxTypesSupported: ['did:ethr:', SubjectIdentifierType.DID],
```

Comment:
did:ethr comes from nowhere ;)

Reply:
no, it was in the previous version with a little change:
```js
didMethodsSupported: ['did:ethr:'],
subjectIdentifiersSupported: SubjectIdentifierType.DID,
```

Reply:
Yeah it was there for demo purposes

Reply:
Is a test class so probably there for a reason